src/app/core/services/issue.service.ts

```diff
@@ -87,15 +87,14 @@ export class IssueService {
           return EMPTY;
         }
         return this.githubService.fetchIssueGraphql(issueId).pipe(
-            map((response) => {
-              const issue = this.createIssueModel(response);
-              this.updateLocalStore(issue);
-              return issue;
-            }),
-            catchError((err) => this.getIssue(issueId))
-          );
-        }
-      )
+          map((response) => {
+            const issue = this.createIssueModel(response);
+            this.updateLocalStore(issue);
+            return issue;
+          }),
+          catchError((err) => this.getIssue(issueId))
+        );
+      })
```

Comment:
This did not actually change any code (only formatting). Thus, this file should not be included in this commit.